### PR TITLE
Update README for `--combinedupgrade` option

### PR DESCRIPTION
**Pull request by thePanz**
See #554

### CODE REVIEW

**Comment by Morganamilo** [comment]
You should probably note that when using `--combinedupgrade`: native pacman is partial

**Comment by thePanz** [comment]
Can you elaborate on `native pacman is partial`? just to mention if there are any pitfalls in using such option :)

**Comment by Morganamilo** [comment]
It's defined here: https://wiki.archlinux.org/index.php/AUR_helpers

I touch on the topic in the commit message but I expect most people to just glance at the faq without clicking the the commit link.

File: README.md [hunk 1]
```diff
@@ -86,6 +86,11 @@ Currently yay Depends on:
 
 * Yay does not display colored output. How do I fix it?
   Make sure you have the `Color` option in your `/etc/pacman.conf` [#123](https://github.com/Jguer/yay/issues/123)
+  
+* Yay is not asking to select packages to install (issue [#554](https://github.com/Jguer/yay/issues/554))
+
+  The default behavior was changed from [v8.918](https://github.com/Jguer/yay/releases/tag/v8.918) (see: [3bdb534](https://github.com/Jguer/yay/commit/3bdb5343218d99d40f8a449b887348611f6bdbfc)).
+  To restore such behavior use the `--combinedupgrade` option, example `yay -Suy --combinedupgrade`, note that in this case the usege of the native `pacman` is partial.
```
Comment (Morganamilo) [inline README.md]: I'd probably prefer `. Note: this causes (link to wiki)[native pacman] to become partial`

You can also rebase all the commits together when you're done.

**Comment by thePanz** [comment]
@Morganamilo patch updated, added note and the link

File: README.md [hunk 2]
```diff
@@ -86,6 +86,12 @@ Currently yay Depends on:
 
 * Yay does not display colored output. How do I fix it?
   Make sure you have the `Color` option in your `/etc/pacman.conf` [#123](https://github.com/Jguer/yay/issues/123)
+  
+* Yay is not asking to select packages during install (issue [#554](https://github.com/Jguer/yay/issues/554))
+
+  The default behavior was changed from [v8.918](https://github.com/Jguer/yay/releases/tag/v8.918) (see: [3bdb534](https://github.com/Jguer/yay/commit/3bdb5343218d99d40f8a449b887348611f6bdbfc)).
+  To restore such behavior use the `--combinedupgrade` option, example `yay -Suy --combinedupgrade`.
+  Note: this causes the [native pacman](https://wiki.archlinux.org/index.php/AUR_helpers) to become partial.
```
Comment (Morganamilo) [inline README.md]: Just to be picky, drop the `the`.

**Comment by Morganamilo** [comment]
You should use `git commit --amend` to amend a commit instead of a generating a new one.

After the fact you can use an interactive rebase with the fixup option,

**Comment by thePanz** [comment]
@Morganamilo I am changing the lines from the Github web-ide, not from a "real" clone. Can you squash-merge this PR?

**Comment by Morganamilo** [comment]
I could squash merge although it'll probably generate a commit message of "Update readme" 4 times over.

Plus I prefer people learning the right way to do things. I'll take your excuse this time.

**Pull request closed without merging**